presto-docs/src/main/sphinx/presto-cpp.rst

```diff
@@ -10,6 +10,7 @@ Note: Presto C++ is in active development. See :doc:`Limitations </presto_cpp/li
     presto_cpp/features
     presto_cpp/limitations
     presto_cpp/properties
+    presto_cpp/properties-session
```

Comment:
Do you want to name the files config-properties.rst and session-properties.rst instead and link from this page ?

Reply:
I thought about that! I thought it would be better to add the new page as properties-session.rst because I didn't want to change the existing page's name from properties.rst. 

This isn't as important for this PR because there's few links into presto_cpp/properties, but because I'm doing this same work for the Java doc in #23811 I didn't want to break the large number of links inbound to admin/properties.rst.